charts/coredns/coredns-pod-delete/engine.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-coredns
+  namespace: kube-system
+spec:
+  appinfo:
+    appns: kube-system
+    applabel: 'k8s-app=kube-dns'
+    appkind: deployment
+  # It can be infra only
+  chaosType: 'infra'
+  #ex. values: ns1:name=percona,ns2:run=nginx 
+  auxiliaryAppInfo: ""
+  components:
+    runner:
+       image: "litmuschaos/chaos-executor:1.0.0"
+       type: "go"
```

Comment:
Replace double quotes to single quotes in every file

Reply:
But it is in docs

Reply:
I have picked these yamls from docs, so should I not pick them from there??